code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -957,10 +959,15 @@
 	add_hiddenprint(user)
 	log_combat(user, src, "[enabled ? "enabled" : "disabled"]")
 	updateTurrets()
-
+/obj/machinery/turretid/proc/shoot_silicons(mob/user)
+	shoot_cyborgs = !shoot_cyborgs
+	add_hiddenprint(user)
+	log_combat(user, src, "[shoot_cyborgs ? "Shooting Borgs" : "Not Shooting Borgs"]")
+	updateTurrets()
 /obj/machinery/turretid/proc/updateTurrets()
 	for (var/obj/machinery/porta_turret/aTurret in turrets)
 		aTurret.setState(enabled, lethal)
+		aTurret.shoot_cyborgs = shoot_cyborgs

```

Comment:
you should change the setState proc to handle cyborgshooting too

Reply:
Dont think it works with that, might.